pikaday-directive.js

```diff
@@ -29,6 +29,20 @@ angular.module('pikaday', [])
                 }
             };
 
+            // Adds weekday/weekend class to cells
+            // Pikaday hides render implementation, need to consider forking it
```

Comment:
Thanks @XmmmiR. Good research. I suppose forking `pikaday-angular` may be something in our future, then.